cmd/jag/commands/flash.go

```diff
@@ -5,18 +5,108 @@
 package commands
 
 import (
+	"bufio"
 	"bytes"
 	"fmt"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"strconv"
+	"strings"
 
 	"github.com/google/uuid"
 	"github.com/spf13/cobra"
 	"github.com/toitlang/jaguar/cmd/jag/directory"
 )
 
+// Returns position, size of the partitions in partitions.csv
+func getPartitions(toitToolchainPath string, positions map[string]int, sizes map[string]int) {
```

Comment:
I'd be curious to see how this would look like if you use encoding/csv (https://pkg.go.dev/encoding/csv).